hack/make-rules/test-cmd.sh

```diff
@@ -58,7 +58,8 @@ function run_kube_apiserver() {
     --cert-dir="${TMPDIR:-/tmp/}" \
     --service-cluster-ip-range="10.0.0.0/24" \
     --token-auth-file=hack/testdata/auth-tokens.csv 1>&2 &
-  APISERVER_PID=$!
+  # Export APISERVER PID
```

Comment:
Sorry if I'm not being clear, I'm asking for you to add these comments above the function, not above the export line, eg:
```
# Runs kube-apiserver
#
# Exports:
#   APISERVER_PID
function run_kube_apiserver() {
```

Reply:
nice!